<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorTab</class>
 <widget class="QWidget" name="MapEditorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>594</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QScrollArea" name="editScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="MapEditorWidget" name="editorWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>574</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rightLayoutWidget">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>Width:</string>
           </property>
           <property name="buddy">
            <cstring>widthSpin</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>Height:</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="buddy">
            <cstring>heightSpin</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="formatLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Format:</string>
           </property>
           <property name="buddy">
            <cstring>formatCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="formatCombo"/>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="widthSpin"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="heightSpin"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="tilesetLabel">
           <property name="text">
            <string>Tileset:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="tilesetButton"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="paletteListWidget">
         <property name="styleSheet">
          <string notr="true">
         QListWidget {
             color: white;
             background-color: #404040;
             font-size: 8pt;
         }
         QListWidget::item {
             padding-top: 2px;
             padding-bottom: 2px;
             background-color: #404040;
         }
         QListWidget::item:selected {
             background-color: green;
         }
        </string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>false</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>false</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapEditorWidget</class>
   <extends>QWidget</extends>
   <header>editor/map/MapEditorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editScrollArea</tabstop>
  <tabstop>widthSpin</tabstop>
  <tabstop>heightSpin</tabstop>
  <tabstop>formatCombo</tabstop>
  <tabstop>tilesetButton</tabstop>
  <tabstop>paletteListWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
